docker/kotlin.docker

```diff
@@ -0,0 +1,58 @@
+FROM codewars/base-runner
```

Comment:
Lets reuse the `java.docker` image for now (eventually we will call this jvm, once we fully replace the existing one). 

At some point managing different versions may be an issue with Kotlin + Java, but for now having to manage a runner container instance for each JVM language we support is going to cost us more RAM and pre-warm/startup time on the server.